fv3config/_tables.py

```diff
@@ -99,30 +84,33 @@ def get_current_date_from_config(config):
     if force_date_from_namelist:
         current_date = config['namelist']['coupler_nml'].get('current_date', [0, 0, 0, 0, 0, 0])
     else:
-        coupler_res_filename = os.path.join(get_initial_conditions_directory(config), 'coupler.res')
+        coupler_res_filename = os.path.join(input_directory, 'coupler.res')
```

Comment:
What was the motivation for passing the input directory separate from config?

Reply:
The code needs to check if `coupler.res` exists among the initial conditions files. I didn't want this part of the code to have to deal with downloading from google cloud storage, so the easier option is to pass the local rundir/INPUT, since the initial conditions files have already been linked or copied to that directory when this code is called. Not a perfect solution, but this can be improved if/when we change the data structure for the initial conditions to a class.

Reply:
Could you add a space-separated sentence with "The input directory is passed to read date from any initial conditions files, since config can contain a remote data source and we do not want to download data in this function."? You can edit it if you want. This should clarify for any future askers of this question.

Reply:
Yup. Added that sentence (with minor edits) to the docstring.